### PR TITLE
chore: bump theme version

### PR DESCRIPTION
**Pull request by CatsJuice**
To remove button's inner shadow

### CODE REVIEW

**Comment by CatsJuice** [comment]
* **#7097** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7097?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CatsJuice and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "@fal-ai/serverless-client": "^0.10.0",
     "@floating-ui/dom": "^1.6.5",
     "@sgtpooki/file-type": "1.0.1",
-    "@toeverything/theme": "^0.7.29",
+    "@toeverything/theme": "^0.7.30",
```
Comment (fourdim) [inline packages/blocks/package.json:26]: The latest commit is not included. Did you forget to release a new one?

Comment (CatsJuice) [reply packages/blocks/package.json:26]: Now I've re-synced with `gt sync`, seems no conflict.

Comment (fourdim) [reply packages/blocks/package.json:26]: I mean the latest commit in @toeverything/theme https://github.com/toeverything/design/commit/04adbc241d39bc2e8e7b521695a5153527c84c38

Comment (CatsJuice) [reply packages/blocks/package.json:26]: This commit currently has no impact on functionality. The main purpose of bumping the theme is to unset the button's inner shadow.

**Review by fourdim — APPROVED** [review]

**Pull request merged**